server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/directives/TestControlStatements.java

```diff
@@ -305,12 +305,16 @@ private static Stream<Arguments> listDirectives() {
             Arguments.of("RULES(ENDPERIOD,EVENPACK,LAXPERF,SLACKBYTES)"),
 
             Arguments.of("NOSSRANGE"),
+            Arguments.of("SSRANGE"),
+            Arguments.of("SSRANGE()"),
```

Comment:
I am not sure I am checking the right thing, but
```
PP 5655-EC6 IBM Enterprise COBOL for z/OS  6.4.0 P240820                
Invocation parameters:                                                  
 SSRANGE()                                                              
                                                                        
        IGYOS4003-E   Invalid option "SSRANGE()" was found and discarded
```

Reply:
![image](https://github.com/user-attachments/assets/8dc18214-bb6b-4803-bfe2-09e5e65a42cf)

Oh, well.